app/data/models/Card.py

```diff
@@ -1,7 +1,17 @@
+from app.logs.logger import logger
+
+
 class Card:
 
     def __init__(self, card_id: int, deck_id: int, word: str, translation: str):
         self.id = card_id
         self.deck_id = deck_id
         self.word = word
         self.translation = translation
+
+        logger.debug(
+            f"Card with id:{self.id}, deck_id:{self.deck_id}, word:{self.word}, translation{self.translation} has been created")
+
+    def __del__(self):
+        logger.debug(
```

Comment:
remove or trace
(requested that already)

Reply:
don't have trace and don't want remove it :/

Reply:
oke